flink-core/src/main/java/org/apache/flink/api/common/ExecutionConfig.java

```diff
@@ -612,6 +614,41 @@ public boolean hasGenericTypesDisabled() {
 		return disableGenericTypes;
 	}
 
+	/**
+	 * Enables the Flink runtime to auto-generated UID's for operators.
+	 *
+	 * @see #disableAutoGeneratedUIDs()
+	 */
+	public void enableAutoGeneratedUIDs() {
+		disableAutoGeneratedUid = false;
+	}
+
+	/**
+	 * Disable's auto-generated UID's forces users to manually specify UID's
+	 * on their datastreams using {@code DataStream#uid} or {@code DataStream#setUidHash}.
+	 *
+	 * <p>It is highly recommended that user's specify UID's before deploying to
+	 * production since they are used to match state in savepoints to operators
+	 * in a job. Because auto-generated ID's are likely to change when modifying
+	 * a job, specifying custom ID's allow an application to evolve overtime
+	 * without discarding state.
+	 */
+	public void  disableAutoGeneratedUIDs() {
+		disableAutoGeneratedUid = true;
+	}
+
+	/**
+	 * Checks whether auto generated UID's are supported.
+	 *
+	 * <p>Auto generated UID's are enabled by default.
+	 *
+	 * @see #enableAutoGeneratedUIDs()
```

Comment:
do we need to change `@see #enableAutoGeneratedUUIDs()` -> `@see {@link #enableAutoGeneratedUUIDs()}`

Reply:
A little confused by what your asking. `@see` automatically creates a link to the method if thats what you mean.